libs/ui/src/components/CommentForm/types.ts

```diff
@@ -0,0 +1,16 @@
+import { User } from '@wsvvrijheid/types'
+export type CommentFormFieldValues = {
+  email: string
+  name: string

```

Comment:
```suggestion
  email?: string
  name?: string
```

email and name should be optional as we don't need for authenticated users.

Reply:
Do we're accepting comments without name or email ?